docker-build.sh

```diff
@@ -1,16 +1,38 @@
-set -ex
+#!/bin/bash
 
-git submodule update --init --recursive
 
-mkdir -p build
+BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
+GIT_REPO="ungoogled-chromium"
 
-# Build first stage docker
-pushd docker-stage1
-docker build --rm -f "Dockerfile" -t ug-chromium-builder-stage1:latest ./
-popd
+DISTRO_RELEASE=${1:-'debian:bookworm'}
+LLVM_VERSION=${2:-'16'}
+NODE_VERSION=${3:-'18'}
 
-# Build second stage docker
-docker build --rm -f "Dockerfile-stage2" -t ug-chromium-builder-stage2:latest ./
+DISTRO=$(echo ${DISTRO_RELEASE}| cut -d':' -f1)
+RELEASE=$(echo ${DISTRO_RELEASE}| cut -d':' -f2)
+[ "$LLVM_VERSION" -gt "16" ] || REPO_POSTFIX="-$LLVM_VERSION"
+
+IMAGE="chromium-builder-${RELEASE}:llvm-${LLVM_VERSION}"
+
+cd $BASE_DIR 
+
+echo "docker buildx build -t ${IMAGE} --build-arg DISTRO=${DISTRO} --build-arg RELEASE=${RELEASE} --build-arg LLVM_VERSION=${LLVM_VERSION} --build-arg REPO_POSTFIX=${REPO_POSTFIX} --build-arg NODE_VERSION=${NODE_VERSION} ."
+(cd $BASE_DIR/docker && docker buildx build -t ${IMAGE} --build-arg DISTRO=${DISTRO} --build-arg RELEASE=${RELEASE} --build-arg LLVM_VERSION=${LLVM_VERSION} --build-arg REPO_POSTFIX=${REPO_POSTFIX} --build-arg NODE_VERSION=${NODE_VERSION} . )
```

Comment:
nit: can use `set -x` before this line and `set +x` after this line so just this line is printed without needing an explicit `echo` command

It took me a minute to realize you were just echoing the command lol.